src/common/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_library(common STATIC common.c)
```

Comment:
Why do you need to build a library instead of just adding common.c to the source list of the tester?